### DISCUSSION TITLE
Change font / rich formatting for text?

### DISCUSSION THREAD

**Discussion by abhijit86k** [xournalpp/xournalpp#3397] [Q&A] [answered]
Is there a way to change the Font and add special formatting (e.g. subscript/superscript) when using the text tool?
TIA

**Comment by Technius** [comment, marked as answer]
You can change the font and font size by selecting the text and clicking the button shown here:
![image](https://user-images.githubusercontent.com/1066652/133863778-aeaefe13-8600-491b-bfdb-79cc12d11dfb.png)

Rich text is not supported directly as of now (relevant feature request: #729), but it is possible to [use the LaTeX tool to insert rich text](https://xournalpp.github.io/guide/tools/latex/). For example:
![image](https://user-images.githubusercontent.com/1066652/133863894-062d964a-fa16-426f-af6c-46782fc38d69.png)

**Reply by abhijit86k** [reply]
Thanks!
I did find a menu entry under Tools>Text Font (Ctrl+Shift+F) which works.
Turns out I was using a toolbar view that does not have it (Right hand note taking). Once I switched o "Xournal++" I got the text selector to show up. Also, I tried "View>toolbars>Customise" but I can't add the font selector to the earlier toolbar view.

I'm running this version:
```
$ xournalpp --version
Xournal++ 1.1.0
└──libgtk: 3.24.25
``

**Reply by Technius** [reply]
If you select "Right hand Note Taking" and then use `View > Toolbars > Customize`, a dialog will pop up asking if you want to copy it. Click yes and then the toolbar customizer will show up; you should then be able to drag the `Font` item to a toolbar.
